src/components/Accordion/docs/AccordionPropsTable.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { PropsTable } from '../../../docs/PropsTable';
+
+const props = [
+    {
+        name: 'heading',
```

Comment:
Please update the props to match the actual API of the component, so the documentation is up-to-date!

- [ ] Update the documentation component to reflect actual API

Reply:
Done